plugins/scaffolder/src/routes.ts

```diff
@@ -16,7 +16,7 @@
 import { createRouteRef } from '@backstage/core';
 
 export const rootRoute = createRouteRef({
-  path: '/create',
+  path: '/create/docs-template',
```

Comment:
Hey! This doesn't seem right. The issue https://github.com/backstage/backstage/issues/4104 is about adding a new button to `plugins/techdocs` which takes the user to the Scaffolder page, and automatically selects a default docs template.

The implementation is non-trivial I would say, since we need to figure out how to define the default docs template.

Reply:
![image](https://user-images.githubusercontent.com/43814313/107663721-b369d200-6cb1-11eb-8066-0b8a691340bc.png)

I guess these are the templates available, which one is default? I will route to it

Reply:
@OrkoHunter  Can we a drop-button?
Create Docs Component -> option 1) Create (Default)  option 2) Choose Template
This will give the user the option to choose and no third button will be used.

Reply:
The "Documentation Template" is the one that we would want to use here - but not every Backstage app will have this exact same template. Some company will use their own docs template, and as of now there is no way to classify templates as Docs template - Everything is a Software template

https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/sample-templates/docs-template/template.yaml

The dropdown idea is good - but we'll still need a filter for "docs template" otherwise there could be dozens of software templates to choose from.

@emmaindal Thoughts? This issue is turning out to be harder than we thought initially. 😬

Reply:
... maybe incremental?  i.e.,

* [ ] Add a create button which just goes to the entire create component page as the first improvement, and includes the support button requested (the bones if you will)
* [ ] Enhance that button to direct link to a configured documentation template(s)

Reply:
Of course Adam to the rescue. I like the idea - let's do the first option in this PR @Anshultilondiya i.e. redirect to `/create` page from the button on TechDocs.

Reply:
Looks like I need to understand the project and its components well, I'm confused 😶

Reply:
There are some demo videos which are a great introduction - https://backstage.io/demos and lots of docs https://backstage.io/docs/overview/what-is-backstage

Will close this PR but feel free to open a new one (do create a new branch out of `master` for the Pull Request) :)

Reply:
Hey, sorry for not responding here sooner. I did not see the notification. My initial thinking was - button will take the user to the docs-template. If that is not configured, fall back to /create. 

And I'm happy to just have a button that takes the user to the /create page as a first step as @adamdmharvey mention if its difficult to know whether the docs-template is used or not.